kedro-datasets/RELEASE.md

```diff
@@ -1,9 +1,17 @@
-# Upcoming Release:
+# Upcoming Release
+## Major features and improvements
+
+## Bug fixes and other changes
+
+## Community contributions
+
+# Release 1.5.0
 
 ## Major features and improvements
 * Implemented lazy loading of dataset subpackages and classes.
     * Suppose that SQLAlchemy, a Python SQL toolkit, is installed in your Python environment. With this change, the SQLAlchemy library will not be loaded (for `pandas.SQLQueryDataSet` or `pandas.SQLTableDataSet`) if you load a different pandas dataset (e.g. `pandas.CSVDataSet`).
 * Added automatic inference of file format for `pillow.ImageDataSet` to be passed to `save()`.
+* Added `DeltaTableDataSet`.
```

Comment:
Maybe worth to mention `pandas`, there are two `DeltaTableDataSet` the old spark version and this new implementation. This allows reading Delta with less dependencies.